packages/react-dom-bindings/src/events/ReactDOMEventReplaying.js

```diff
@@ -105,7 +111,7 @@ const discreteReplayableEvents: Array<DOMEventName> = [
   'change',
   'contextmenu',
   'reset',
-  'submit',
+  // 'submit', // stopPropagation blocks the replay mechanism
```

Comment:
This is really unfortunate and I'm not sure what the correct solution is.

We currently call `stopPropagation` when we might have hydrated some of the tree but not all the way to the target. We do this because we want to treat the tree as if it wasn't hydrated at all. Meaning we shouldn't call any parent event handlers if they would've been stopped.

However, this is not like "before" hydration for any other more global scripts that were inserted early or by third parties.

In this case, this blocks our event replaying handler at the root so we don't call preventDefault and it just ends up going through the `javascript:` URL.

Reply:
What event handlers is it trying to block? Native event handlers added by components would be added by useEffect which hasn't run at this point right?

Reply:
It could've been if it's in a parent component with a Suspense/Hydration boundary around the target.